### DISCUSSION TITLE
Contact Form & Product Reviews

### DISCUSSION THREAD

**Discussion by blueportal31415** [saleor/saleor#6024] [Q&A] [answered]
Hello,

Is there a way to add "product reviews"  & "Contact form" functions ?

**Comment by korolevpavel** [comment]
Hello @blueportal31415 !
I think now this can only be done through the Pages

**Reply by blueportal31415** [reply]
@korolevpavel Have you implemented a contact form via Pages before?

**Comment by karolkielecki** [comment]
@blueportal31415 Hi there! 

Regarding the `product reviews` feature - we have it on our roadmap :)

**Reply by blueportal31415** [reply]
Nice @karolkielecki !  Good to know.

**Reply by tusharsrivastava** [reply]
Hi @karolkielecki could you just guide me (provide basic pointers) as to how I can implement a simple contact form on an application. I'm already thanking you for the help 😇😇😇

**Reply by karolkielecki** [reply]
Could you please provide more details regarding your use case?

**Comment by timuric** [comment, marked as answer]
We are not planning to add product reviews, however, you could use product metadata to provide very basic reviews or use a full fledge service for reviews such as [trustpilot ](https://www.trustpilot.com/) and integrate it with Saleor. 
For contact forms please consider using other tools on the market, we don't consider this to be an essential feature for an ecommerce platform.
